README.md

```diff
@@ -31,14 +69,393 @@ Permissions to allow the user to interact with the support center:
+
+### 3.2.4. Status
+
+Updates the Account CR
+```
+claimed: false
+conditions:
+- lastProbeTime: 2019-07-18T22:04:38Z
+  lastTransitioNTime: 2019-07-18T22:04:38Z
+  message: Attempting to create account
+  reason: Creating
+  status: "True"
+  type: Creating
+rotateCredentials: false
+state: Failed
+supportCaseID: "00000000"
+```
+
+*state* can be any of the account states defined in the constants
+*claimed* is true if `currentAcctInstance.Status.State == AccountReady && currentAcctInstance.Spec.ClaimLink != "`
+*rotateCredentials* when true rotates the temporary credentials on the next run
+*supportCaseID* is the ID of the aws support case to increase limits
+*conditions* indicates the last state the account had and supporting details
+
+### 3.2.5. Metrics
+
+Update in the account-controller
+```
+MetricTotalAWSAccounts
+```
+
+## 3.3. AccountClaim Controller
+
+The accountClaim-controller is triggered when an accountClaim is created in any namespace. It is responsible for following behaviours:
+
+1. Sets account `spec.ClaimLink` to the name of the accountClaim
+2. Sets accountClaim `spec.AccountLink` to the name of an unclaimed Account 
+3. Creates a secret in the accountClaim namespace that contains the credentials tied to the aws account in the accountCR
+4. Sets accountClaim `status.State = "Ready"` 
+
+### 3.3.1. Constants and Globals
+```
+AccountClaimed          = "AccountClaimed"
+AccountUnclaimed        = "AccountUnclaimed"
+awsCredsUserName        = "aws_user_name"
+awsCredsAccessKeyId     = "aws_access_key_id"
+awsCredsSecretAccessKey = "aws_secret_access_key"
+```
+
+### 3.3.2. Spec
+
+Updates the accountClaim CR
+```
+spec:
+  accountLink: osd-{accountName} 
+  aws:
+    regions:
+	- name: us-east-1
+  awsCredentialSecret:
+	 name: aws
+     namespace: {NameSpace}
+   legalEntity:
+	 id: 00000000000000
+	 name:{Legal Entity Name}
+```
+
+*awsCredentialSecret* holds the name and namespace of the secret with the credentials created for the accountClaim
+
+### 3.3.3. Status
+
+Updates the accountClaim CR
+```
+conditions:
+  - lastProbeTime: 2019-07-16T13:52:02Z
+    lastTransitionTime: 2019-07-16T13:52:02Z
+	message: Attempting to claim account
+	reason: AccountClaimed
+	status: "True"
+	type: Unclaimed
+  - lastProbeTime: 2019-07-16T13:52:03Z
+    lastTransitionTime: 2019-07-16T13:52:03Z
+	message: Account claimed by osd-creds-mgmt-fhq2d2
+	reason: AccountClaimed
+	status: "True"
+	type: Claimed
+  state: Ready
+```
+
+*state* can be any of the ClaimStatus strings defined in accountclaim_types.go
+*conditions* indicates the last state the account had and supporting details
+
+### 3.3.4. Metrics
+
+Updated in the accountClaim-controller
+```
+MetricTotalAccountClaimCRs
+```
+
+# 4. Special Items in main.go
+
+* Initailzes a watcher that will look at all the secrets in the accountCRNamespace and if it exceeds the const `secretWatcherScanInterval` it will set the corresponsing accountCR `status.rotateCredentials = true`
```

Comment:
accountCRNamespace? spaces?

the `secretWatcherScanInterval` just set this the interval at which the secret watcher will look for secrets that are expiring. See the logic for when we renew these secrets here https://github.com/openshift/aws-account-operator/blob/master/pkg/credentialwatcher/secretwatcher.go#L85